src/index.js

```diff
@@ -50,9 +50,9 @@ const onfidoRender = (options, el, merge) => {
   return render( <Container options={options}/>, el, merge)
 }
 
-
-const stripOneCapture = ({image,documentType,id}) =>
-  documentType === undefined ? {id,image} : {id,image,documentType}
+const stripOneCapture = ({image,documentType,id}) => {
```

Comment:
Yes, I was meant to put it back as it was after the changing I did on the function in older commits. Will do it now